lib/govuk_app_config/govuk_data_sync.rb

```diff
@@ -0,0 +1,31 @@
+require "time"
+
+class GovukDataSync
+  def initialize(govuk_data_sync_period)
+    @from, @to = false
+    begin
+      parts = govuk_data_sync_period&.split("-")
+      @from, @to = parts.map { |time| Time.parse(time) } if parts&.count == 2
+    rescue ArgumentError
+      # At least one of the parts was malformed. Leave @from/@to as false.
+    end
```

Comment:
I feel like we might want this to blow up on a malformed env var rather than gracefully handle it. I'm imagining that if we messed up the env var we'd have very little clues about it aside from overnight when the errors started coming in. Whereas if it blew up quickly you'd get much faster feedback.